src/Indicator.vala

```diff
@@ -37,14 +37,18 @@ public class Session.Indicator : Wingpanel.Indicator {
     private Session.Services.UserManager manager;
 
     private Gtk.Grid main_grid;
-    private Session.Widgets.EndSessionDialog? shutdown_dialog = null;
 
     public Indicator (Wingpanel.IndicatorManager.ServerType server_type) {
         Object (code_name: Wingpanel.Indicator.SESSION,
                 display_name: _("Session"),
                 description: _("The session indicator"));
         this.server_type = server_type;
         this.visible = true;
+
+        if (server_type == Wingpanel.IndicatorManager.ServerType.SESSION) {
```

Comment:
I'm unsure about this because I think we want to be able to use shortcuts to shutdown/restart on the greeter right?